code/modules/mob/living/simple_animal/borer/borer.dm

```diff
@@ -233,5 +262,84 @@
 	var/datum/ghosttrap/G = get_ghost_trap("cortical borer")
 	G.request_player(src, "A cortical borer needs a player.", ANIMAL)
 
+/mob/living/simple_animal/borer/proc/borer_add_exp(var/num)
+	borer_exp += num
+	update_borer_level()
+
+/mob/living/simple_animal/borer/proc/update_borer_level()
+	if((borer_exp >= 20) && (borer_level < 1))
+		borer_level = 1
+		produced_reagents |= list("inaprovaline", "tricordrazine", "synaptizine", "imidazoline", "hyronalin")
+		abilities_in_host |= list(/mob/living/simple_animal/borer/proc/say_host, /mob/living/simple_animal/borer/proc/whisper_host, /mob/living/simple_animal/borer/proc/commune)
+		abilities_standalone |= list(/mob/living/simple_animal/borer/proc/commune)
+		if(host && !controlling)
+			verbs += /mob/living/simple_animal/borer/proc/say_host
+			verbs += /mob/living/simple_animal/borer/proc/whisper_host
+			verbs += /mob/living/simple_animal/borer/proc/commune
+		if(!host)
+			verbs += /mob/living/simple_animal/borer/proc/commune
+		to_chat(src, SPAN_NOTICE("Congratulations! You've reached Evolution Level 1, new syntesis reagents and new abilities are now available."))
```

Comment:
As you've copypasted this each time, might be an idea to break the function into different procs.

Also, "synthesis", not "syntesis"

Reply:
Fixed.